src/validations/javascript/jshint.js

```diff
@@ -21,6 +21,18 @@ const jshintrc = {
   undef: true,
 };
 
+const jshintrcExternalScript = {
```

Comment:
doing it this way is unsustainable in the long run, but I figured beats having mutable configuration for now?